### ISSUE TITLE
Cannot be used in a md-dialog

### ISSUE THREAD

**Issue by ViDaNgEr** [logbon72/angular-material-datetimepicker#25]
The angular-material-datetimepicker cannot be used in a md-dialog because md-dialog cannot be nested yet.

https://github.com/angular/material/issues/698

**Comment by patrickmichalina** [comment]
Yea :(

**Comment by ViDaNgEr** [comment]
We can close this one.  Fixed with PR #27

**Comment by johhansantana** [comment]
How to use it?

I currently have it like this:

``` html
<input mdc-datetime-picker
date="true"
time="true"
type="text"
id="time"
short-time="true"
placeholder="Time"
format="MM-DD-YYYY HH:mm"
skipHide="true"
ng-model="newNote.start">
```

and it closes my `md-dialog` after clicking on it.

**EDIT**
Well, it appears that the bower version doesn't have this feature yet. It works when I cloned this repository.

**Comment by mandadimuralidharreddy** [comment]
@jsantana90  you should bower master branch  then it will resolve.

**Comment by alobban** [comment]
skipHide isn't working. Was this feature released into the wild on v1.4.0?
